include/taskolib/Step.h

```diff
@@ -98,6 +98,9 @@ class Step
      *                      - A message of type step_stopped_with_error when the step has
      *                        been stopped due to an error condition
      * \param index         Index of the step in its parent Sequence.
+     * \param step_setup    Guarantees the execution of the setup script before the
```

Comment:
Maybe just "A setup script that is executed just before the main script of the step"? The user can see the default parameter by herself.